modules/sharethroughBidAdapter.js

```diff
@@ -113,7 +113,7 @@ export const sharethroughAdapterSpec = {
 
     req.user = nullish(firstPartyData.user, {});
     if (!req.user.ext) req.user.ext = {};
-    req.user.ext.eids = bidRequests[0].userIdAsEids || [];
+    req.user.ext.eids = bidRequests?.[0]?.user?.ext?.eids || bidRequests[0].userIdAsEids || [];
```

Comment:
```suggestion
    req.user.ext.eids = bidderRequest.ortb2.user?.ext?.eids || [];
```
(`bidRequests[0].ortb2.user.ext.eids` would also work, but both need the `.ortb2`)

https://github.com/prebid/Prebid.js/pull/13535 should make it so that `.userIdAsEids` is equivalent, but using first party data is preferable - and looking at both is unnecessary even before that gets merged.